### DISCUSSION TITLE
Timelines: Controlling sample size and performance concerns

### DISCUSSION THREAD

**Discussion by chipmasters** [uwrit/leaf#596] [Q&A]
Hi Leaf Developers,

We are working to release the Timelines feature in our Production Leaf environment at Icahn School of Medicine, and we have had a request to increase the sample size from the default 500 patients to a larger value. Is it possible to change that setting in the configuration files? Has anyone tested using larger values to determine how it impacts performance?

Thanks,
James "Chip" Masters

**Comment by ndobb** [comment]
Hi Chip,

Sure - the sample size used in Timelines is the same as that used for Patient List, specifically the [`Cohort:ExportLimit`](https://github.com/uwrit/leaf/blob/master/src/server/API/appsettings.json#L70) value in your production appsettings.json file.

After changing the value, you'll need to restart the Leaf API for changes to take effect.

How high are you considering changing the value to? Others, such as the University of Colorado @Ram-SN, if I recall correctly switched to a much higher limit, though I don't know how high or what performance issues (if any) they encountered.

Best,
-nic
